src/state/migrations.ts

```diff
@@ -0,0 +1,39 @@
+import { createMigrate, MigrationManifest, PersistedState, PersistMigrate } from 'redux-persist';
+import { MigrationConfig } from 'redux-persist/lib/createMigrate';
+
+function parseStorageItem(data: string | null) {
+  if (!data) return undefined;
+
+  try {
+    return JSON.parse(data);
+  } catch (_) {
+    return undefined;
+  }
+}
+
+export function customCreateMigrate(
+  migrations: MigrationManifest,
+  options: MigrationConfig
+): PersistMigrate {
+  const defaultMigrate = createMigrate(migrations, options);
+  return async (state: PersistedState, currentVersion: number) => {
+    if (state !== undefined) {
```

Comment:
so where do real migrations go?

Reply:
GOOD question i can include an example in this PR!

Reply:
updated! PTAL